packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchor.razor.cs

```diff
@@ -10,6 +10,12 @@ public partial class NimbleAnchor : NimbleAnchorBase
     [Parameter]
     public bool? UnderlineHidden { get; set; }
 
+    /// <summary>
+    /// Whether the anchor should behave like it is in an editable region.
+    /// </summary>
+    [Parameter]
+    public bool? ContentEditable { get; set; }
```

Comment:
> My impression is that we can take more liberties with the types in the Blazor wrappers 

I prefer to not have arbitrary inconsistencies between all the different framework wrappers. The goal someday would be to generate as much as possible and this would not be a high value exception to that case with custom logic.

Also this is a browser native API, not one we are inventing. You can use all the other HTML elements in a Blazor App. Ours should not have arbitrary differences. They are HTML Elements and this is a standard HTML Element API.